backend/shoulder/s2s/views.py

```diff
@@ -1611,12 +1616,37 @@ def trigger_suggestions(self, request):
         if update_response.status_code != 200:
```

Comment:
Should the predictions be updated each time a user suggestion get request is made? I think we came to the conclusion that updates should only happen on a weekly-ish basis, and instead suggestions should be taken directly from the EventSuggestions table for this endpoint

Reply:
I think instead line 1609/1614 should be `update_response = SuggestionResults.objects.get(user_id=user_id)`

Reply:
Also, as part of the response for the `trigger_suggestions()` function, can you also include how far the event is from the user? You should be able to do this by modifying lines 1663-1675 and adding something like `event_suggestion['distance_from_user'] = next(key for key, value in self.distance_calc(event_id, user_id).items() if value)` which gets the first key where the distance within is true; so if the returned dictionary is 
```python
{'dist_within_1mi': False,
 'dist_within_5mi': True,
 'dist_within_10mi': True,
 'dist_within_15mi': True,
 'dist_within_20mi': True,
 'dist_within_30mi': True,
 'dist_within_40mi': True,
 'dist_within_50mi': True}
```

then the code would return 'dist_within_5mi'

Reply:
That's right! will implement

Reply:
Re distance, also implementing!